modmail/database/models/__init__.py

```diff
@@ -0,0 +1,20 @@
+from .attachments import Attachments
```

Comment:
Should these all be plural or singular? Not sure of the convention.

Reply:
The convention I have used so far and seen so far is keeping model names as plural.